src/jira/test.js

```diff
@@ -0,0 +1,33 @@
+const config = require("../config");
+const notifyConfig  = config.get("notify");
+var JiraClient = require('jira-connector');
+
+var jira = new JiraClient( {
+    host: notifyConfig.jira.host,
+    basic_auth: {
+        base64: notifyConfig.jira.basic_auth.base64
+    }
+});
+
+var issue = {
+    "fields": {
+        "project":{
+            "key": notifyConfig.jira.project_key
+        },
+        "summary": "Test from Schnack",
+        "description": "Congratulations, you have successfully configured Schnack to create a new task when a comment is posted.\n\nNice work :-)",
+        "issuetype": {
+            "name": notifyConfig.jira.issue_type
+        }
+    }
+}
+jira.issue.createIssue(issue, function callback(no, data, response) {
```

Comment:
What's `no` as a value? Could this have a clearer name?

Reply:
The `no` variable was recommended by the NPM Package; but you are correct, it's not terribly descriptive. This variable is **always** `null`; so I've renamed it to `empty`.